File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/AzeritePower.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AzeritePower {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("spell")
    @Expose
    public Spell spell;
    @SerializedName("tier")
    @Expose
    public Integer tier;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/AzeritePower.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class AzeritePower(
    @SerializedName("id")
    @Expose
    var id: Int? = null,

    @SerializedName("spell")
    @Expose
    var spell: Spell? = null,

    @SerializedName("tier")
    @Expose
    var tier: Int? = null
)